src/coreclr/jit/ssabuilder.cpp

```diff
@@ -1388,9 +1388,19 @@ void SsaBuilder::AddPhiArgsToSuccessors(BasicBlock* block)
 #if defined(TARGET_WASM)
         }
 #endif
+
+
         // Now handle memory.
         for (MemoryKind memoryKind : allMemoryKinds())
         {
+#ifdef TARGET_WASM
+            // TODO-LLVM: LIR memory liveness is NYI upstream. Delete when that is fixed.
+            if (block->IsLIR())
+            {
+                return;
```

Comment:
```suggestion
                break;
```
(C&P bug on my part)

Reply:
ah, yes I perhaps should have spotted that.  Thanks